doc/landing-page/index.html

```diff
@@ -0,0 +1,259 @@
+                            <div class="icon-box">
+                                <div class="icon-box-inner">
+                                    <span data-feather="git-pull-request" width="28" height="28"></span>
+                                </div>
+                            </div>
+                            <div class="media-body">
+                                REPL - to quickly and easily interact with ASA and Smart Contracts
+                            </div>
+                        </div>
+                    </div><div class="col-sm-6">
+                        <div class="media">
+                            <div class="icon-box">
+                                <div class="icon-box-inner">
+                                    <span data-feather="zap" width="28" height="28"></span>
+                                </div>
+                            </div>
+                            <div class="media-body">
+                                Runtime - to test smart contracts very fast without being on the real blockchain
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<!--Features Section-->
+<div class="container">
+    <div class="row">
+        <div class="col-md-9 mx-auto">
+            <h2 class="text-center text-md-left">We are waiting on your feedback, contact us using algo-builder
+                <a href="https://github.com/scale-it/algo-builder/issues">
+                    <span class="text-secondary">GitHub Issues</span></a>
+            </h2>
+        </div>
+    </div>
+</div>
+
+<!--footer-->
+<footer class="py-5 bg-light">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 text-center">
+                <ul class="list-inline">
+                    <li class="list-inline-item"><a href="https://github.com/scale-it/algo-builder/issues">Contact Us</a></li>
+                    <li class="list-inline-item"><a href="../algob/index.html">API Docs</a></li>
+                    <li class="list-inline-item"><a href="https://github.com/scale-it/algo-builder">Fork algo-builder on GitHub</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="row my-2">
+            <div class="col-md-4 mx-auto text-muted text-center small-xl">
+                &copy; 2020 Algo Builder - All Rights Reserved
+            </div>
+        </div>
+    </div>
+</footer>
+
+<!-- jQuery first, then Popper.js, then Bootstrap JS -->
+<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/feather-icons/4.7.3/feather.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/prism.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/plugins/line-numbers/prism-line-numbers.min.js"></script>
```

Comment:
:+1: for using CDN

Reply:
Maybe we can use the latest version of Prism? It's `1.23.0`
https://github.com/PrismJS/prism/releases

Reply:
> Maybe we can use the latest version of Prism? 

Yes, thanks. Updated prism version to `1.23.0`. Also updates feather-icons cdn version to `4.28.0`.

Reply:
Edit: Updated prism version to `1.17.0`, after that ":" in code was disappearing (with the current theme)